osf/metadata/utils.py

```diff
@@ -0,0 +1,86 @@
+from website import settings
+
+
+SUBJECT_SCHEME = 'bepress Digital Commons Three-Tiered Taxonomy'
+
+RESOURCE_TYPE_MAP = {
```

Comment:
Since this is specific to datacite, should consider renaming `DATACITE_RESOURCE_TYPE_MAP` or moving to the `DataciteRecordSerializer`